todoapp/users/urls.py

```diff
@@ -1,10 +1,10 @@
-from django.conf.urls import url
-from users.views import UserRegistrationAPIView, UserLoginAPIView, UserLogoutAPIView
+from django.urls import path
+from users.views import UserRegistrationAPIView, UserLoginAPIView, UserTokenAPIView
 
 app_name = 'users'
 
 urlpatterns = [
-    url(r'^$', UserRegistrationAPIView.as_view(), name="list"),
-    url(r'^login/$', UserLoginAPIView.as_view(), name="login"),
-    url(r'^logout/$', UserLogoutAPIView.as_view(), name="logout"),
+    path(r'users/', UserRegistrationAPIView.as_view(), name="list"),
+    path(r'users/login/', UserLoginAPIView.as_view(), name="login"),
+    path(r'tokens/<pk>/', UserTokenAPIView.as_view(), name="token"),
```

Comment:
@rigaspapas Thank you for your contribution. 

You are passing token `key` value with URL but the parameter naming defined as `pk`. This namespace can cause confusion. Can you use `key` instead of `pk` here?

Reply:
The rest of the PR is fine.

Reply:
You are right @erdem. The `pk` is the expected keyword argument for the `RetrieveDestroyAPIView`, so I changed it to `key` and I also set the `lookup_field = "key"` class property. I force-pushed my changes.

Reply:
Looking good @rigaspapas, thank you for your contribution again!